src/util/input.rs

```diff
@@ -16,7 +16,13 @@ pub fn process_mouse(event: MouseEvent, context: &mut AppContext, backend: &mut
     let constraints: &[Constraint; 3] = &context.config_ref().display_options_ref().default_layout;
     let layout_rect = Layout::default()
         .direction(Direction::Horizontal)
-        .vertical_margin(1)
```

Comment:
This fixed the border-bug by providing the correct view port height and the correct y-origin for the entry lists.